apps/law_memory/observables.py

```
@@ -0,0 +1,18 @@
+"""Utilities for converting BEC observables into semantic fields."""
+
+from __future__ import annotations
+
+import numpy as np
+
+
+def density_to_gamma(density: np.ndarray) -> np.ndarray:
+    """Convert a condensate density array into the γ coherence field."""
```

Ruff (0.13.3)

Docstring contains ambiguous `γ` (GREEK SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF002)